tools/python/influx-migration/README.md

```diff
@@ -114,7 +114,7 @@ The following is a formatted version of the output:
 
 **`--dest-bucket DEST_BUCKET`**: Optional. The name of the InfluxDB bucket in the destination server, must not be an already existing bucket. Defaults to value of `--src-bucket` or `None` if `--src-bucket` not provided.
 
-**`--dest-host DEST_HOST`**: The host for the destination server. Example: http://localhost:8086.
+**`--dest-host DEST_HOST`**: The host for the destination server. Must have a scheme, domain or IP address, and port, e.g., `http://127.0.0.1:8086` or `https://<domain>:<port>`.
```

Comment:
If the `dest-host` points to a Timestream InfluxDB instance, then the scheme can have only `https:` as the scheme and supports only `domain` and not IP.

Reply:
`dest-host` can also be an InfluxDB instance, which supports using an IP instead of a domain and `http` as a scheme.